kotlin/src/main/kotlin/org/jetbrains/research/testspark/kotlin/KotlinPsiHelper.kt

```diff
@@ -121,19 +112,13 @@ class KotlinPsiHelper(private val psiFile: PsiFile) : PsiHelper {
 
         repeat(maxInputParamsDepth) {
             val tempListOfClasses = mutableSetOf<KotlinPsiClassWrapper>()
-
             currentLevelClasses.forEach { classIt ->
                 classIt.methods.forEach { methodIt ->
                     (methodIt as KotlinPsiMethodWrapper).parameterList?.parameters?.forEach { paramIt ->
-                        val typeRef = paramIt.typeReference
-                        if (typeRef != null) {
-                            resolveClassInType(typeRef)?.let { psiClass ->
-                                if (psiClass.kotlinFqName != null) {
-                                    KotlinPsiClassWrapper(psiClass as KtClass).let {
-                                        if (!it.qualifiedName.startsWith("kotlin.")) {
-                                            interestingPsiClasses.add(it)
-                                        }
-                                    }
+                        KtPsiUtil.getClassIfParameterIsProperty(paramIt)?.let { typeIt ->
+                            KotlinPsiClassWrapper(typeIt).let {
+                                if (!it.qualifiedName.startsWith("java.")) {
+                                    interestingPsiClasses.add(it)
```

Comment:
Shouldn't it be a `.startsWith("kotlin.")` or both?

Reply:
Sure, should be Kotlin